cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1398,12 +1406,12 @@ def default_for(
                    {REPLACES}``global.rc[hosts][<host>]copyable
                    environment variables``.
             ''')
-            Conf('retrieve job logs', VDR.V_BOOLEAN, desc=f'''
-                .. versionchanged:: 8.0.0
-
-                   {REPLACES}``global.rc[hosts][<host>]retrieve job logs``.
-                   {PLATFORM_REPLACES.format("[remote]retrieve job logs")}
-            ''')
+            Conf(
+                'retrieve job logs', VDR.V_BOOLEAN,
```

Comment:
Why isn't there a default value for this (`True` or `False`)?

Reply:
Because history I'm afraid. We should make these defaults explicit - https://github.com/cylc/cylc-flow/issues/4847